spec/support/custom_matchers/have_attributes.rb

```diff
@@ -16,6 +16,8 @@
 
         matcher = if actual.respond_to?(:acts_like_time?) && expected.respond_to?(:acts_like_time?)
                     be_same_time_as(expected)
+                  elsif expected.kind_of?(RSpec::Matchers::BuiltIn::Match)
+                    match(expected.expected)
```

Comment:
Does it make sense to deprecate this custom matcher in a future release so we finish converting to the rspec one?

What steps do we need to take to use the rspec version and get off of our custom one?